controllers/provider-openstack/charts/internal/machineclass/templates/machineclass.yaml

```diff
@@ -27,7 +27,11 @@ spec:
   availabilityZone: {{ $machineClass.availabilityZone }}
   flavorName: {{ $machineClass.machineType }}
   keyName: {{ $machineClass.keyName }}
+{{- if $machineClass.imageID }}
```

Comment:
Can you add `imageID` to the `values.yaml` (commented) so that it's clear which values are possible (https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-openstack/charts/internal/machineclass/values.yaml#L7)?

```yaml
  ...
  keyName: my-ssh-key
  imageName: coreos-v1.0
# imageID: uid-of-image
  ...
```